### PR TITLE
[libvirt] Capture virtlogd.conf

### PR DESCRIPTION
**Pull request by dnegreira**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/report/plugins/libvirt.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def setup(self):
             "/etc/libvirt/storage/autostart/*.xml",
             "/etc/libvirt/qemu-lockd.conf",
             "/etc/libvirt/virtlockd.conf",
+            "/etc/libvirt/virtlogd.conf"
```
Comment (jcastill) [inline sos/report/plugins/libvirt.py:45]: you'll probably get an alert from one of the tests for the missing comma here

**Review by jcastill — COMMENTED** [review]
I was going to ask for the need to obfuscate stuff in this config file but I think is covered down the file already

File: sos/report/plugins/libvirt.py (hunk 1 above)
Comment (dnegreira) [reply sos/report/plugins/libvirt.py]: whoops, fixed now.

**Comment by dnegreira** [comment]
> I was going to ask for the need to obfuscate stuff in this config file but I think is covered down the file already

There isn't any sensitive data in this file anyway

**Review by arif-ali — APPROVED** [review]

**Review by pmoravec — APPROVED** [review]

**Pull request merged**